doc/ld65.sgml

```diff
@@ -562,7 +562,8 @@ segment, where this attribute is true, the linker will export three symbols.
 </verb></tscreen>
 
 Replace <tt/NAME/ by the name of the segment, in the example above, this would
-be <tt/BSS/. These symbols may be accessed by your code.
+be <tt/BSS/. These symbols may be accessed by your code once the symbol has
+been imported using the <tt>.IMPORT</tt> directive.
```

Comment:
"Symbols" is plural.  Therefore, it should say, "`... once they have been imported ...`".

Also, it might be a good idea to say, "`... using ca65's <tt>.IMPORT</tt> directive.`"